app/components/created-by.js

```diff
@@ -11,8 +11,8 @@ export default Component.extend({
   }),
 
   showUser: computed('nameOrLogin', 'eventType', function () {
-    let nameOrLogin = this.get('nameOrLogin');
-    let eventType = this.get('eventType');
+    let nameOrLogin = this.nameOrLogin;
+    let eventType = this.eventType;
```

Comment:
Another curiosity, I saw that we use `let` instead of `const` throughout the codebase, is there any reason for that?

Reply:
I don't think we have any firm policy on use of `let` or `const`, and a lot of people have worked on `travis-web` over the years. I think in many cases it was someone's preference at the time, but there are probably also a few cases where there was a reason for it. But there's no single overall reason for it that I'm aware of.